src/tree/AttachedServersTreeItem.ts

```diff
@@ -105,17 +132,97 @@ export class AttachedServersTreeItem implements IAzureParentTreeItem {
     private async loadPersistedServers() {
         const value: any = this._globalState.get(this._serviceName);
```

Comment:
Error handling?  I.e. settings are corrupt.  Or they have a later version of the extension then backgrade to this version, and the format has changed.